cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -570,6 +582,9 @@ def start_master():
     configure_controller_manager()
     configure_scheduler()
 
+    # make all services restart all the time
+    add_systemd_restart_always()
```

Comment:
This should probably go before `configure_apiserver()`, `configure_controller_manager()`, `configure_scheduler()` since that's when those services are restarted.